### PR TITLE
[US100303] Build DB catalog service

### PR DESCRIPTION
**Pull request by duyphatbk**
Add backend catalog

### CODE REVIEW

File: apps/backend/catalog-ms/src/main/resources/application.properties [hunk 1]
```diff
@@ -1,2 +1,9 @@
-server.port=80
-spring.devtools.restart.enabled = true 
\ No newline at end of file
+server.port=8090
```
Comment (LittleLegend-Pengiun) [inline apps/backend/catalog-ms/src/main/resources/application.properties]: You should update the port in terraform as well.

File: apps/backend/catalog-ms/src/main/java/com/service/catalog/reposistory/CategoryReposistory.java [hunk 2]
```diff
@@ -0,0 +1,8 @@
+package com.service.catalog.reposistory;
+
+import com.service.catalog.models.Category;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+public interface CategoryReposistory extends JpaRepository<Category, String> {
```
Comment (LittleLegend-Pengiun) [inline apps/backend/catalog-ms/src/main/java/com/service/catalog/reposistory/CategoryReposistory.java]: why there are empty class here?

**Review by LittleLegend-Pengiun — CHANGES_REQUESTED** [review]

File: apps/backend/catalog-ms/src/main/resources/application.properties (hunk 1 above)
Comment (duyphatbk) [reply apps/backend/catalog-ms/src/main/resources/application.properties]: updated

File: apps/backend/catalog-ms/src/main/java/com/service/catalog/reposistory/CategoryReposistory.java (hunk 2 above)
Comment (duyphatbk) [reply apps/backend/catalog-ms/src/main/java/com/service/catalog/reposistory/CategoryReposistory.java]: It's standard

**Review by LittleLegend-Pengiun — DISMISSED** [review]

**Review by LittleLegend-Pengiun — APPROVED** [review]

**Pull request merged**